packages/client/modules/meeting/components/AddPokerTemplateDimension.tsx

```diff
@@ -0,0 +1,84 @@
+import styled from '@emotion/styled'
+import graphql from 'babel-plugin-relay/macro'
+import React, {Component} from 'react'
+import {createFragmentContainer} from 'react-relay'
+import {Threshold} from '~/types/constEnums'
+import Icon from '../../../components/Icon'
+import LinkButton from '../../../components/LinkButton'
+import withAtmosphere, {
+  WithAtmosphereProps
+} from '../../../decorators/withAtmosphere/withAtmosphere'
+import AddPokerTemplateDimensionMutation from '../../../mutations/AddPokerTemplateDimensionMutation'
+import dndNoise from '../../../utils/dndNoise'
+import withMutationProps, {WithMutationProps} from '../../../utils/relay/withMutationProps'
+import {AddPokerTemplateDimension_dimensions} from '../../../__generated__/AddPokerTemplateDimension_dimensions.graphql'
+
+const AddDimensionLink = styled(LinkButton)({
+  alignItems: 'center',
+  display: 'flex',
+  justifyContent: 'flex-start',
+  fontSize: 16,
+  lineHeight: '24px',
+  margin: 0,
+  marginBottom: 16,
+  outline: 'none',
+  padding: '8px 0'
+})
+
+const AddDimensionLinkPlus = styled(Icon)({
+  display: 'block',
+  margin: '0 16px 0 16px'
+})
+
+interface Props extends WithAtmosphereProps, WithMutationProps {
+  dimensions: AddPokerTemplateDimension_dimensions
+  templateId: string
+}
+
+class AddPokerTemplateDimension extends Component<Props> {
```

Comment:
nit: can you write this as a functional component?
we deprecated class-based components back in 2018 & have been slowly trying to get them out of the app. Once they're gone we can get rid of a lot of class-only helpers like `withAtmosphere` so we can keep our codebase as small as possible